test/userbufferedmessage.cpp

```diff
@@ -45,7 +47,7 @@ static void test_req(const char *data,
   while (used < datalen) {
     used += parser.feed(data + used, datalen - used);
   }
-  parser.feed((void*)NULL, 0);
+  // parser.feed((void*)NULL, 0);
```

Comment:
Why did you remove this?  Also, if we need to remove it, maybe remove the line instead?